tests/utils.py

```diff
@@ -9,3 +11,7 @@ def assertNotRaises(self, exc_type):
             yield None
         except exc_type:
             raise self.failureException("{} raised".format(exc_type.__name__))
+
+
+class WidgetTestForm(forms.Form):
```

Comment:
I found this quite strange since it says WidgetTestForm and then doesn't contain a widget! Consider whether you actually need this in test/utils.py IMO you could easily just define this class inside of the test as you're only using this form in one test and are unlikely to use it elsewhere.